acos_client/v30/slb/service_group.py

```diff
@@ -50,19 +50,26 @@ def get(self, name, **kwargs):
         return self._get(self.url_prefix + name, **kwargs)
 
     def _set(self, name, protocol=None, lb_method=None, hm_name=None,
-             update=False, **kwargs):
+             hm_disable=0, update=False, **kwargs):
```

Comment:
since you're changing the method signature slightly, did you grep everywhere to make sure you caught all usages of this?